src/Aspire.Dashboard/Components/Controls/GridValue.razor.css

```diff
@@ -9,19 +9,23 @@
     grid-template-columns: 1fr auto auto;
 }
 
-::deep .cellText {
+::deep .grid-value {
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
 }
 
+::deep .grid-value > .exception-details-button {
```

Comment:
IMO add `exception-details-button` styles to global app.css.

The component isn't specific to grid value - it could be placed anywhere - it just happens to only be used in grid value at the moment.

Reply:
Further changes made this rule unnecessary.